2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -4,14 +4,14 @@
 MAX_ATTEMPTS=60
 SLEEP_TIME=1
 
-export MONGODB_CONFIG_PATH=/var/lib/mongodb/mongodb.conf
-export MONGODB_PID_FILE=/var/lib/mongodb/mongodb.pid
-export MONGODB_KEYFILE_PATH=/var/lib/mongodb/keyfile
+export MONGODB_CONFIG_PATH=/etc/mongod.conf
+export MONGODB_PID_FILE=${HOME}/mongodb.pid
```

Comment:
we were avoiding ${HOME} and rather use the absolute path in case you use different user and the HOME can be set to something unpredictable.